CONTRIBUTING.md

```diff
@@ -1,41 +1,49 @@
 
 ### Bug Reporting Guidelines
-* __Important__: Report the issue using our [template][template], it includes all the
-  information we need to track down the issue.  
-* __SECURITY__: Report any potential security bug by following our [security policy](https://nextcloud.com/security/) instead of filing an issue in our bug tracker
+* __Important__: [Complete the bug report form](https://github.com/nextcloud/desktop/issues/new/choose). If you fill it correctly, it should give us enough information to start working on the issue.
+* __SECURITY__: **Please do _not_ report security vulnerabilities through public GitHub issues.** 
+If you have discovered a security matter with Nextcloud, please first read our [responsible disclosure guidelines](https://nextcloud.com/security/) and our [security policy](https://github.com/nextcloud/server/blob/9459724e72d647bb21d0bc36c6dc56b751771130/SECURITY.md). Then contact us at [hackerone.com/nextcloud](https://hackerone.com/nextcloud).
 * This repository is *only* for issues within the Nextcloud desktop client. 
   Issues in other components should be reported in their own repositores: 
   - [Nextcloud server](https://github.com/nextcloud/server/issues)
   - [Android client](https://github.com/nextcloud/android/issues)
   - [iOS client](https://github.com/nextcloud/ios/issues)
 * Search the existing issues first, it's likely that your issue was already
   reported.
-
-If your issue appears to be a bug, and hasn't been reported, open a new issue.
-
-Help us to maximize the effort we can spend fixing issues and adding new
-features, by not reporting duplicate issues.
-
-[template]: https://raw.githubusercontent.com/nextcloud/desktop/master/.github/issue_template.md
-[support]: https://nextcloud.com/support/
-[forum]: https://help.nextcloud.com/categories
-[irc]: https://webchat.freenode.net/?channels=nextcloud
+* If your issue appears to be a bug and hasn't been reported, open a new issue.
+  Help us to maximize the effort we can spend fixing issues and adding new features, by not reporting duplicate issues.
 
 ## Contributing to Source Code
-
 Thanks for wanting to contribute source code to Nextcloud. That's great!
 
-You do not need to sign a Contributor Agreement, but we ask that you follow our 
-[Code of Conduct](https://nextcloud.com/code-of-conduct/).
+We ask that you follow our [Code of Conduct](https://nextcloud.com/code-of-conduct/).
 
 Please read the [Contribution Guide](https://nextcloud.com/contribute/) to get 
 started and follow the [Coding Style](https://github.com/nextcloud/desktop/wiki/Coding-Style)
 when writing new code.
 
+### Sign your work
+
+We use the Developer Certificate of Origin (DCO) as an additional safeguard
+for the Nextcloud project. This is a well established and widely used
+mechanism to assure contributors have confirmed their right to license
+their contribution under the project's license.
+Please read [contribute/developer-certificate-of-origin][dcofile].
```

Comment:
The link reference to `dcofile` is missing

we can just link directly to the server repo:

```suggestion
Please read [contribute/developer-certificate-of-origin](https://github.com/nextcloud/server/blob/master/contribute/developer-certificate-of-origin).
```